draft-ietf-rats-architecture.md

```diff
@@ -805,6 +814,8 @@ to convey unprotected Evidence, assuming the following properties exists:
    2. All unprotected Evidence that is conveyed is supplied exclusively by the Attesting Environment that has the key  material that protects the conveyance channel
    3. The Root of Trust protects both the conveyance channel key material and the Attesting Environment with equivalent strength protections.
 
+See {{security-considerations}} for discussion on security strength.
+
```

Comment:
The anchor for {{security-considerations}} needs to be added.

Reply:
I took the liberty and addressed the comment directly in this PR. Please find the added label below.